README.md

```diff
@@ -101,6 +115,7 @@ network_manager_stopall="stop-all"
 network_manager_restart="restart"
 network_manager_home_profile="home"
 network_manager_work_profile="work"
+network_manager_other_profiles="other-profiles"
```

Comment:
```suggestion
network_manager_other_profiles="another-profile, yet-another-profile, last-profile"
```

As discussed, to stress that the option is a CSV list, this could help.

Reply:
96ce0f6 fixes this. 418880d updates README contents to reflect changes.